website/package.json

```
@@ -7,9 +7,10 @@
   },
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "docusaurus start",
-    "build": "yarn generate && docusaurus build",
+    "start": "yarn fetch-remote-content && docusaurus start",
+    "build": "yarn generate && yarn fetch-remote-content && docusaurus build",
     "swizzle": "docusaurus swizzle",
+    "fetch-remote-content": "docusaurus download-remote-content-external && docusaurus download-remote-content-sdks",
     "generate": "docusaurus gen-api-docs all && node clean-generated-docs.js",
     "deploy": "yarn generate && docusaurus deploy",
```

we probably also need it in `deploy`, since it's the script used in `generate-docs.yaml` CI file

---

Ah, very good catch!

```suggestion
    "deploy": "yarn generate && yarn fetch-remote-content && docusaurus deploy",
```